src/controller/genericmidi.cxx

```diff
@@ -623,28 +398,30 @@ void GenericMIDI::launchScene( int scene )
     
     if ( b->action == GRID_LAUNCH_SCENE )
     {
-      for( int i = 0; i < 5; i++ )
```

Comment:
well, you can test this one with apc, if it has also 8 scene launch buttons like mine. this change with the loop doesn't effect the color